howtos/stable_kernel_backport.rst

```diff
@@ -19,12 +19,14 @@ please indicate if you have applied these changes.
 Install all updates before following these instructions.
 And test your system again after a reboot to ensure that this procedure is still necessary.
 
-As of writing, the Leap 15.3 used in our v4 "Built on openSUSE" comes with many openSUSE/SuSE organised patches.
+As of writing, the Leap version used in our v4 "Built on openSUSE" comes with many openSUSE/SuSE organised patches.
 Almost all of these patches are backports from newer kernels,
-applied to a designated 'base' kernel version of, in the case of Leap 15.3, **5.3.18**.
+applied to a designated 'base' kernel version.
+As such the running kernel is newer than its 'base' version indicates.
 
-Rockstor no longer installs newer kernels than it's openSUSE/SuSE upstream, as it once did when based on CentOS 7 (Rockstor v3).
-That is primarily because it is no longer necessary to do so,
+Rockstor V4 "Built on openSUSE" no longer installs newer kernels than it's upstream OS,
```

Comment:
Minor typo:
```suggestion
Rockstor V4 "Built on openSUSE" no longer installs newer kernels than it's upstream OS,
```
```suggestion
Rockstor V4 "Built on openSUSE" no longer installs newer kernels than its upstream OS,
```

Reply:
Thanks @FroggyFlox I really do have a little problem there. Funnily enough I nearly added a new one also :) , but your prior corrections came to mind.